cmd/gtk/assets/ui/main_window.ui

```diff
@@ -103,6 +103,33 @@
                   <object class="GtkMenu">
                     <property name="visible">True</property>
                     <property name="can-focus">False</property>
+                    <child>
+                      <object class="GtkMenuItem" id="id_website_menu">
+                        <property name="visible">True</property>
+                        <property name="can-focus">False</property>
+                        <property name="label" translatable="yes">Website</property>
```

Comment:
You can update the code as follows:
```
<property name="label" translatable="yes">_Website</property>
```
Adding an underscore (_) before "Website" sets "W" as the accelerator key for the menu item. (Alt+W)